SimpleWAWS.Console/Program.cs

```diff
@@ -3,6 +3,7 @@
 using System;
```

Comment:
`SimpleWAWS.Console` is a test project for easier testing of the api. not sure why this would be needed here. This project never runs in prod.